tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -81,6 +82,7 @@ def test_arxiv_fulltext_download_logs_on_success():
     assert expected == result
 
 
+@pytest.mark.xfail(raises=DownloadError)
```

Comment:
No it says that I expect to fail with an exception = DownloadError. It is like asserting that there is an exception.